examples/fluent-bit/send-to-multiple-destinations/extra.conf

```diff
@@ -10,4 +10,26 @@
     Match  *
     region us-west-2
     delivery_stream stream-two
-    retry_limit 2
\ No newline at end of file
+    retry_limit 2
+
+[OUTPUT]
+    Name cloudwatch_logs
+    Match *
+    region ${AWS_REGION}
+    log_key log
+    log_group_name /aws/ecs/${ECS_CLUSTER}
+    log_stream_name ${ECS_TASK_ID}
```

Comment:
Sorry I didn't think of this before, the env vars here are only valid if we use init. And so either you need to remove this OR:

1. Change the task def to use the image `docker pull public.ecr.aws/aws-observability/aws-for-fluent-bit:init-latest`: https://github.com/aws-samples/amazon-ecs-firelens-examples/blob/mainline/examples/fluent-bit/send-to-multiple-destinations/task-definition.json#L8
2. Add a note in the readme that this example uses the init tag for ECS Metadata and link the ECS metadata init example: https://github.com/aws-samples/amazon-ecs-firelens-examples/tree/mainline/examples/fluent-bit/init-metadata